### DISCUSSION TITLE
大家都有搶到周杰倫的票嗎？

### DISCUSSION THREAD

**Discussion by seastar0687** [1101-GS4719-C/Python-Community-GS4719-C#2207] [General]
今天看到的限動不是在等搶票就是正在搶票，我自己是對周杰倫沒有過多的興趣，爾且搶票時間是12.，要認真上家凱的課，所以沒有搶。
事實上對於周杰倫原來在我們的世代這麼熱門有一點意外，大家怎麼看呢？
順便問問大家都有搶到票嗎？
還是都在認真上課呢？

**Comment by 112121py** [comment]
幫朋友搶的，上課前就已經開好輸入完驗證碼了(我有上課!!)，結果那個圈圈轉了20分鐘然後跟我說沒了q。
我覺得周董在我們這代還很熱門感覺也蠻正常的(?，雖然隨著時代的演進，大家已經不只有台灣的歌手可以聽，還有很多像K-POP、J-POP等等的文化進來，讓聽歌這件事情變得非常多元，但原本就喜歡周董的人感覺也還是會喜歡啦~

**Comment by poyen0806** [comment]
今天上課前，就有很多人在教室搶票，其中還包含了上節課的老師。這個時候就有一個值得討論的問題：在這間教室搶票，網路真的有比較快嗎？這間教室在電算中心旁，所以可以較快的將電腦的request送到電算中心server，再由server交出去，最後到搶票網站的server，但其中的過程我們卻不知道發生什麼，只知道有人網站卡住搶不到票。這可能是中間的過程出了什麼問題，也可能單純網站server扛不住大量的購票資訊導致卡頓，我認為這個問題值得分析，不知道其他人是什麼看法
